src/helpers/constants.js

```diff
@@ -6,7 +6,9 @@ export const HELP_REQUEST = 'Help Request';
 export const WELFARE_CALL = 'Welfare Call';
 export const ALL = 'All';
 export const LINK_WORK = 'Link Work';
+export const EUSS = 'EUSS';
 export const DEFAULT_DROPDOWN_OPTION = 'Please choose';
+export const IS_EUSS_ENABLED = false;
```

Comment:
We could change this to.. 

export const IS_EUSS_ENABLED = process.env.NODE_ENV !== 'production';

.. so it will make it visible / testable in Staging.